### DISCUSSION TITLE
没有基础视频教程（配置拼音输入），门槛太高了。直接劝退百分之90的普通用户。

### DISCUSSION THREAD

**Discussion by pros2021** [osfans/android-rime#1426] [General | 常规讨论]
一个基础的视频教程也就几分钟而已，就能解决大部分问题了。

要把视频教程做好、做清楚。但是现在的情况是：全网都找不到完整可靠的视频教程。。。

电脑、手机的这个开源输入法，最大的缺点就是没有基础视频教程，直接劝退百分之90的普通用户。很明显这是个程序员思维，没有产品经理辅助、导致没有从用户角度着想 完善教程。

希望完善基础 基础的视频教程就行了！或者内置拼音输入和九宫格。 
软件安装好之后，至少得能直接输入拼音，这是基本的。其他小众的人想折腾，就去折腾。

**Comment by MokOopsing** [comment]
https://github.com/osfans/trime/wiki

rime及各个前端是开发者用爱发电的项目，你用不用它，开发者没有任何好处和损失，（相信）它不是为你们这些连wiki都不看的人设计的！

**Comment by pros2021** [comment]
> https://github.com/osfans/trime/wiki
> 
> rime及各个前端是开发者用爱发电的项目，你用不用它，开发者没有任何好处和损失，（相信）它不是为你们这些连wiki都不看的人设计的！

我也知道是开源项目，但是让它越来越好、难道不是你我的责任和义务吗？

你会了，不代表百分之90的小白用户也能折腾到输入拼音。

你学会了，是否更应该贡献出自己的一份力、把基础的教程做好，造福更多因为门槛高而弃坑的百分之90的用户。

我真诚的邀请你，做一个几分钟的视频教程，就是安卓手机怎么配置拼音输入的教程。 希望你贡献自己的一份力量。

**Reply by qualvey** [reply]
没有什么责任和义务。你有意愿可以自己做教程。

**Comment by tumuyan** [comment]
一个问题是，不同版本的操作方法是不同的，不同方案和配置文件遇到的问题也各不相同。一个视频并不能解决大部分人无法打字的困境。甚至开发者也经常为无法打字而困扰。
![IMG_20230116_155054](https://user-images.githubusercontent.com/3126801/212625156-8b6c9a2c-7db8-4138-b329-190e9e6a5156.jpg)

虽然现在Android设备已经比Windows设备应用更为广泛，但是还是推荐用户先使用过Windows端小狼毫，并且放弃九宫格键盘的想法再来试用同文。
因为即使抛开前边的困难，同文的九宫拼音还是很难用。

**Comment by MokOopsing** [comment]
> > https://github.com/osfans/trime/wiki
> > rime及各个前端是开发者用爱发电的项目，你用不用它，开发者没有任何好处和损失，（相信）它不是为你们这些连wiki都不看的人设计的！
> 
> 我也知道是开源项目，但是让它越来越好、难道不是你我的责任和义务吗？
> 
> 你会了，不代表百分之90的小白用户也能折腾到输入拼音。
> 
> 你学会了，是否更应该贡献出自己的一份力、配合我把基础的教程做好，造福更多因为门槛高而弃坑的百分之90的用户。
> 
> 我真诚的邀请你，和我做一个几分钟的视频教程，就是安卓手机怎么配置拼音输入的教程。 希望你贡献自己的一份力量。

不要要求别人，要求自己

**Comment by zhaozg** [comment]
+1 to close

**Comment by Meavel** [comment]
https://www.youtube.com/results?search_query=%E5%90%8C%E6%96%87%E8%BE%93%E5%85%A5%E6%B3%95

这里有很多教程……如果不够的话，我也试试看自己录一套吧？

**Comment by pros2021** [comment]
> https://www.youtube.com/results?search_query=%E5%90%8C%E6%96%87%E8%BE%93%E5%85%A5%E6%B3%95
> 
> 这里有很多教程……如果不够的话，我也试试看自己录一套吧？

😅早都看过了，只有那一个客家话讲的是2年前手机输入法。。。而且是只是装上，你再去仔细看一下搜出来的视频。可用的教程没有。
而b站上，多数是哑巴解说，让人摸不到头脑。

**Comment by chaosder** [comment]
争取今年8月我录一个

**Comment by pros2021** [comment]
> 争取今年8月我录一个

已经有入门教程了，【trime同文输入法】配置中文拼音方案的入门解说教程https://www.bilibili.com/video/BV1YA411z7Um/?vd_source=6d42b271f854a930cc01c2c14ad98639

**Comment by pros2021** [comment]
> https://www.youtube.com/results?search_query=%E5%90%8C%E6%96%87%E8%BE%93%E5%85%A5%E6%B3%95
> 
> 这里有很多教程……如果不够的话，我也试试看自己录一套吧？

已经有入门教程了，【trime同文输入法】配置中文拼音方案的入门解说教程https://www.bilibili.com/video/BV1YA411z7Um/?vd_source=6d42b271f854a930cc01c2c14ad98639

**Comment by pros2021** [comment]
> 一个问题是，不同版本的操作方法是不同的，不同方案和配置文件遇到的问题也各不相同。一个视频并不能解决大部分人无法打字的困境。甚至开发者也经常为无法打字而困扰。 ![IMG_20230116_155054](https://user-images.githubusercontent.com/3126801/212625156-8b6c9a2c-7db8-4138-b329-190e9e6a5156.jpg)
> 
> 虽然现在Android设备已经比Windows设备应用更为广泛，但是还是推荐用户先使用过Windows端小狼毫，并且放弃九宫格键盘的想法再来试用同文。 因为即使抛开前边的困难，同文的九宫拼音还是很难用。

已经有入门教程了，【trime同文输入法】配置中文拼音方案的入门解说教程https://www.bilibili.com/video/BV1YA411z7Um/?vd_source=6d42b271f854a930cc01c2c14ad98639

**Comment by RaidOpe** [comment]
别像小丑，开源不代表免费，视频只是手段，不是目的。你可以在知乎，贴吧去提问。而不是应该在提出bug的地方来批判作者和其他用户，推广是重要，但我觉得若你能辅助开发更重要，权利和义务是分不开的

**Comment by StellaMeow** [comment]
电脑端还好，Trime经常性配置失败，如果能有个详细的解说会便于用户使用

**Comment by WhiredPlanck** [comment]
自 https://github.com/osfans/trime/commit/2ce23418cba3306be64f60dfe2edd20f38399ff9 起同文添加了明月拼音和五笔画作为预置方案，增强开箱体验。
